src/main/java/forestry/storage/ModuleCrates.java

```diff
@@ -67,6 +80,105 @@ public void preInit() {
 		MinecraftForge.EVENT_BUS.register(this);
 	}
 
+	@Override
+	public void doInit() {
```

Comment:
You are parsing the config at `doInit` but all creates get created in the `RegistryEvent.Register<Item>` event and that is called directly after `preInit`.